rskj-core/src/test/java/co/rsk/config/RskSystemPropertiesTest.java

```diff
@@ -34,21 +42,59 @@
     public void defaultValues() {
         Assert.assertFalse(config.isMinerClientEnabled());
```

Comment:
I'd use static imports for this as well (so it's the same as the changes you made)

Reply:
done